src/components/home/Team/index.js

```diff
@@ -0,0 +1,75 @@
+import _ from "lodash"
+import Img from "gatsby-image"
+import React from "react"
+
+import { StaticQuery, graphql } from "gatsby"
+
+import "./index.module.css"
+
+const socialLink = (platform, username) => {
```

Comment:
Not that important, but `socialUrl` describes this function better, since we're not actually returning the link.

Reply:
I honestly don't think this matters. Many times we call "link" to "urls".